docs/_data/mesheryctlcommands/cmds.yml

```
@@ -477,5 +477,61 @@ pattern:
         json:
           name: -o json
           description: display the content of a pattern in json format
-          usage: |
-              mesheryctl pattern view [pattern-name|pattern-id] -o json
\ No newline at end of file
+          usage:
+              mesheryctl pattern view [pattern-name|pattern-id] -o json
+
+app:
+  name: app
+  description: Apps management of service meshes
+  usage:
+    mesheryctl app 
+  flags:
+    file:
+      name: --file, -f
+      description: (required) path to pattern file
+      usage:
+        mesheryctl app --file [path to pattern file]
+  subcommands:
+    list:
+      name: list
+      description: displays a list of available applications
+      usage:
+          mesheryctl app list
+    offboard:
+      name: offboard
+      description: offboard application file
+      usage:
+          mesheryctl app offboard [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: Offboard application by providing file path
+          usage:
+              mesheryctl app offboard -f [filepath] 
+    onboard:
+      name: onboard
+      description: onboard application file
+      usage: 
+          mesheryctl app onboard [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: onboard application with application file
+          usage:
+              mesheryctl app onboard --file [filepath]
+    view:
+      name: view
+      description: displays the contents of a specific pattern file
+      usage:
+          mesheryctl app view BookInfo[application-name|application-id] [flags]
```

```suggestion
          mesheryctl app view BookInfo (application-name|application-id)
          OR
          mesheryctl app view BookInfo
```
not sure! but it'll look good;

---

similarly in other areas!

---

+1 to @afzal442 's point. Let's be consistent with other commands here.